packages/ember-htmlbars/tests/compat/view_helper_test.js

```diff
@@ -71,14 +71,15 @@ QUnit.test('using the view helper with a string (inline form) fails assertion [L
   const ViewClass = EmberView.extend({
     template: compile('fooView')
   });
-  registry.register('view:foo', ViewClass);
+  owner.register('view:foo', ViewClass);
 
   ignoreAssertion(function() {
     component = EmberComponent.extend({
-      layout: compile('{{view \'foo\'}}'),
-      container
+      layout: compile('{{view \'foo\'}}')
     }).create();
 
+    setOwner(component, owner);
```

Comment:
should be easy to create a helper that produces it for the extend (like my comment bellow)